### PR TITLE
Links to Explainable embeddings

### PR DESCRIPTION
**Pull request by elboyran**
- [x] Link to the RSD page for the explainable embedding software.
- [x] Update the roadmap.
- [x] Update documentation

### CODE REVIEW

**Comment by elboyran** [comment]
I've also updated the [RSD software page](https://research-software-directory.org/software/dianna).

**Comment by elboyran** [comment]
Updated [documentation link](https://dianna.readthedocs.io/en/link2explainable-embeddings/)

File: README.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ DIANNA supports different data modalities and XAI methods (explainers). We have
...
 | Timeseries | ✅                                                | ✅                                                                 |                                                                                                     |
 | Tabular    | planned                                           | ✅                                                                 | ✅                                                                                                  |
-| Embedding  | work in progress                                  |                                                                     |                                                                                                     |
+| Embedding  | *modified RISE in [Explainable embeddings package](https://github.com/dianna-ai/explainable_embedding)  |                                                                     |                                                                                                     |
```
Comment (cwmeijer) [inline README.md]:
```suggestion
| Embedding  | *inspired by RISE in [distance_explainer](https://github.com/dianna-ai/distance_explainer)  |                                                                     |                                                                                                     |
```

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-- Scientific use-cases: the use of XAI in scientific applicaitons: tutorials and videos
-
-- Other data modalities: we plan support for graphs.
-
-- Embeddings: We plan to add support for embeddings as a data modality in the near future. This will allow users to create the same or similar explanations and visualizations for their models that is projecting data towards embedded spaces. 
+- Other data modalities: we plan to develop support for graphs.
 
 - Add support for text for KernelSHAP XAI method.
 
+- Extend the dashboard with more examples and make it to work smoothly with custom data and models
```
Comment (cwmeijer) [inline docs/ROADMAP.md]:
```suggestion
- Extend the dashboard with more examples and make it work with custom data and models
```

**Review by cwmeijer — CHANGES_REQUESTED** [review]
Thanks for adding the link! I corrected the link to the repo and did a textual suggestion.

File: README.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ The key points of DIANNA:
  *   Implements well-known XAI methods LIME, RISE and KernelSHAP, chosen by systematic and objective evaluation criteria
  *   Comes with a dashboard where  results of different explainers can be compared for all data types
  *   Supports the de-facto standard of neural network models - ONNX
- *   Supports images, text, time series, and tabular data modalities, embeddings are currently being developed
+ *   Supports images, text, time series, tabular data modalities and embeddings (in a related [package](https://github.com/dianna-ai/explainable_embedding))
```
Comment (cwmeijer) [inline README.md:78]:
```suggestion
 *   Supports images, text, time series, tabular data modalities and embeddings (in a related package, [distance_explainer](https://github.com/dianna-ai/distance_explainer))
```

**Review by cwmeijer — APPROVED** [review]
I saw one more link that needed updating to our distance_explainer repo (see suggestion/comment). Please commit that one as well before merging.
Thanks for this PR!

**Pull request merged**